sim/Makefile

```diff
@@ -34,7 +34,7 @@ base_dir := $(firesim_base_dir)
 chipyard_dir := $(abspath ..)/target-design/chipyard
 rocketchip_dir := $(chipyard_dir)/generators/rocket-chip
 
-JVM_MEMORY ?= 16G
+JVM_MEMORY ?= 32G
```

Comment:
really?

Reply:
This was only for some larger designs we were playing with. It's not because of mulitclock. I'll drop it back down.